src/jobsautoreport/main.py

```diff
@@ -36,10 +37,13 @@ def main() -> None:
             microsecond=0,
             tzinfo=timezone.utc,
         )
-        report_start_time = report_end_time - relativedelta(weeks=1)
+        current_report_start_time = (
```

Comment:
why is this formatted this way? isn't this just
`a = b = t - relativedelta(...)` ?
I think we could possibly also remove a bit of boilerplate common code